pkg/setting/setting.go

```diff
@@ -1504,6 +1508,7 @@ func (cfg *Cfg) GetContentDeliveryURL(prefix string) string {
 func (cfg *Cfg) readDataSourcesSettings() {
 	datasources := cfg.Raw.Section("datasources")
 	cfg.DataSourceLimit = datasources.Key("datasource_limit").MustInt(5000)
+	cfg.ResponseLimit = datasources.Key("response_limit").MustInt64(maxResponseSize)
```

Comment:
Think it might make more sense to put the setting under `dataproxy` because we have all other related data source settings there, like timeouts and such. WDYT?

Reply:
Yeah both make sense to me, but let's keep it simple under `[dataproxy]` since it's going to be the whole lot of other datasources as you said 👍